### ISSUE TITLE
Trapezius muscle aplasia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7679]
**Preferred term label:**
Trapezius muscle aplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**
A developmental defect characterized by lack of formation of the trapezius muscle, which is broad superficial muscle that extends from the posterior neck to the posterior part of the trunk (thorax).

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
600257 
PMID:7849713  (no freq)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
